src/modules/board.js

```diff
@@ -1,10 +1,8 @@
-function Board() {
-  const cells = document.querySelectorAll('.cell');
-  const winningMsg = document.querySelector('.winning-msg');
-  const winData = document.querySelector('[data-win-text]');
-  const positions = Array.from(cells);
-  const checkForWinner = () => {
-    let winner = false;
+function Board(positions, winData, winningMsg) {
+  // const cells = document.querySelectorAll('.cell');
+  // const positions = Array.from(cells);
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.

Kindly remove these two lines.